src/main/java/org/folio/rest/impl/InvoiceHelper.java

```diff
@@ -450,7 +451,19 @@ private void setSystemGeneratedData(Invoice invoiceFromStorage, Invoice invoice)
     return completedFuture(null);
   }
 
+  private void verifyApprovalAllowed(Invoice invoiceFromStorage, Invoice invoice) {
```

Comment:
I would suggest renaming this method because the name now doesn't match what it actually does.